tslearn/barycenters/softdtw.py

```diff
@@ -100,7 +100,7 @@ def softdtw_barycenter(X, gamma=1.0, weights=None, method="L-BFGS-B", tol=1e-3,
         barycenter = init
 
     if max_iter > 0:
-        X_ = numpy.array([to_time_series(d, remove_nans=True) for d in X_])
+        X_ = [to_time_series(d, remove_nans=True) for d in X_]
```

Comment:
Here, do we have a test with time series of unequal length somewhere, to make sure that this formulation actually works?

Reply:
OK, so the doctest actually checks this, so if all tests pass, that means this implementation is OK